python/tank/platform/interactive_authentication.py

```diff
@@ -295,28 +299,38 @@ def ui_renew_session():
     """
     Prompts the user to enter his password in a dialog to retrieve a new session token.
     """
-    UiAuthenticationHandler(is_session_renewal=True).authenticate()
+    # Force human user authentication, since ression renewal is always in the context of a user login.
+    UiAuthenticationHandler(is_session_renewal=True).authenticate(force_human_user_authentication=True)
 
 
-def ui_authenticate():
+def ui_authenticate(force_human_user_authentication=False):
     """
-    Prompts the user to login via a dialog and caches the session token for future reuse.
+    Authenticates the current process. Authentication can be done through script user authentication
+    or human user authentication. If doing human user authentication and there is no session cached, a
+    dialgo asking for user credentials will appear.
```

Comment:
If its temporary then no need to change anything! :) I am sure all the pieces will align beautifully later on!